src/Group.php

```diff
@@ -83,5 +121,18 @@ public function getMembers(): array
     {
         return $this->members;
     }
-
+    /**
+     * @param User $user
+     * @return void
+     */
+    public function addUserToGroup($user)
```

Comment:
we know its about adding a user to a group, because we are using the Group class
```suggestion
    public function addUser($user)
```